test/metabase/query_processor_test/cumulative_aggregation_test.clj

```diff
@@ -246,3 +246,47 @@
                   [#t "2016-06-01" 57 3391.41 59.50]]
                  (mt/formatted-rows [->local-date int 2.0 2.0]
                    (qp/process-query query)))))))))
+
+(deftest ^:parallel expressions-inside-cumulative-aggregations-test
```

Comment:
Added a few extra tests. This stuff was already working, but we didn't have tests around these specific things